### DISCUSSION TITLE
Delrin wheels?

### DISCUSSION THREAD

**Discussion by tdurk93** [stevenbennett/DIYson#44] [Q&A] [answered]
I see that your BOM includes the [Xtreme V Wheels](https://openbuildspartstore.com/xtreme-solid-v-wheel-kit/?ref=_2E7m4SXRIy8t). Have you tried using the slightly cheaper Delrin [Solid V Wheels](https://openbuildspartstore.com/solid-v-wheel-kit/?ref=_2E7m4SXRIy8t)? Not sure if we need the increased compression resistance of the Xtreme wheels, or if this was simply a preference of color. With 6 wheels, the $1.80/wheel cost savings would add up to nearly $11.

**Comment by legopc** [comment]
I use the cheapest wheels from Aliexpress and have no issues so there is definitely cost saving-potential. Life-span is another issue though and I cant really comment on that.

**Comment by stevenbennett** [comment, marked as answer]
@tdurk93 I imagine that most v-slot wheels work fine, I just chose the Open Builds ones for predictably and because many other required components can be found there. Builder's choice though, do whatever works for you 👍🏻
